lib/parseSty.js

```diff
@@ -0,0 +1,101 @@
+"use strict";
+
+var starts = ["\\newcommand",
```

Comment:
This variable name is not self explaining. Either rename it, or add a comment what this array is supposed to do.